pycytominer/tests/test_cyto_utils/test_load.py

```diff
@@ -196,3 +198,28 @@ def test_load_npz():
         load_npz_locations(
             example_npz_file_locations, location_x_col_index=0, location_y_col_index=2
         )
+
+def test_is_path_a_parquet_file():
+
+    # file paths
+    test_data_dir = "tests/test_data/cytominer_database_example_data"
+    csv_file = f"{test_data_dir}/test_SQ00014613.csv.gz"
+    parquet_file = f"{test_data_dir}/test_SQ00014613.parquet"
+
+    # checking parquet file
+    check_pass = is_path_a_parquet_file(parquet_file)
+    check_fail = is_path_a_parquet_file(csv_file)
+
+    # checking if the correct booleans are returned
+    assert(check_pass, True)
+    assert(check_fail, False)
+
+    # loading in pandas dataframe from parquet file
+    parquet_df = pd.read_parquet(parquet_file)
+    parquet_profile_test = load_profiles(parquet_file)
+    pd.testing.assert_frame_equal(parquet_profile_test, parquet_df)
+
+    # loading csv file with new load_profile()
+    csv_df = pd.read_csv(csv_file)
+    csv_profile_test = load_profiles(csv_file)
+    pd.testing.assert_frame_equal(csv_profile_test, csv_df)
```

Comment:
Consider adding a newline at the end of this file.

Reply:
Still here